bin/publish

```diff
@@ -19,28 +19,42 @@ ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../"
 CURR_BRANCH="$( git rev-parse --abbrev-ref HEAD )"
 
 # Pull new staging code
+# when testing, use `git checkout origin/staging`
```

Comment:
Not sure if we should include this comment. In theory it should be tested, and if someone can't figure out to switch it to origin when they need to test it, they probably shouldn't be running this script.